Source/.eslintrc.json

```diff
@@ -14,6 +14,7 @@
         "es/no-modules": "off",
         "es/no-typed-arrays": "off",
         "es/no-block-scoped-variables": "off",
-        "es/no-template-literals": "off"
+        "es/no-template-literals": "off",
+        "es/no-promise": "off"
```

Comment:
This can easily be a follow-up PR, but given all of the changes in the past year, we should revisit Cesium's default eslintrc configuration and the shared eslint-cesium plugin we also maintain in this repository. A lof of these settings are out of date ad unneeded (like the es/restrict-to-es5).  It's not a ton of work, but does affect JS code cesium-wide, so I would just write an issue and we can easily bang it out as soon as we get a chance.

Reply:
Definitely. We already planned on addressing eslint configuration in the near future, but I'll write up an issue to be clear about it.